GitCommands/Settings/ConfigFileSettings.cs

```diff
@@ -56,7 +56,7 @@ public static ConfigFileSettings CreateGlobal(ConfigFileSettings? lowerPriority,
             if (!File.Exists(configPath))
             {
                 // Git 1.xx
-                configPath = Path.Combine(AppSettings.GitBinDir, "..", "etc", "gitconfig");
+                configPath = Path.Combine(AppSettings.GitCommand, "..", "..", "etc", "gitconfig");
```

Comment:
Doesn't `AppSettings.GitCommand` contain the path to git.exe?

Reply:
> Doesn't `AppSettings.GitCommand` contain the path to git.exe?

Yes, it does. This setting is correctly named (in contrast to `GitBinDir`). The global settings are located in the Git installation folder.
This is the actual bugfix (done in a separate commit, not to be squashed).